### DISCUSSION TITLE
Gain of RTLSDR

### DISCUSSION THREAD

**Discussion by kochikame** [junzis/pyModeS#159] [Q&A]
Hi,
I would like to know how can I send GAIN commands when we call modeslive --source rtlsdr?
Thanks in advance

**Comment by xoolive** [comment]
The gain setting is automatic, see https://github.com/junzis/pyModeS/blob/77273153cba6c2f282f672ea4078a62efcf716d7/pyModeS/extra/rtlreader.py#L41

Then you may want to send a PR on the modeslive script to take that parameter into account. The script starts in `streamer/modeslive.py:main()`

**Comment by kochikame** [comment]
Thanks a lot
My intention is create a portable TCAS module using a raspberry pico with a sdr and a gps to calculate the proximity with other planes. It will be good when I am piloting a piper PA28 in my air club.
